.github/workflows/ci.yml

```diff
@@ -19,3 +18,8 @@ jobs:
 
       - name: Lint
         run: yarn lint
+
+      - name: Check for Broken Links
+        run: |
+          yarn global add mintlify
+          mintlify broken-links
```

Comment:
this was super helpful locally and helped me catch a bunch of existing broken links we have/had